### PR TITLE
Mobile dedicated gallery widget

### PR DESCRIPTION
**Pull request by ithiame**
Create a mobile-dedicated gallery widget variation #1241

### CODE REVIEW

**Comment by coyotte508** [comment]
IMO we should merge https://github.com/B2Bitcoin/beBOP/pull/1271 first and see what's still needed

File: src/lib/components/GalleryWidget/GalleryWidget.svelte [hunk 1]
```diff
@@ -11,61 +12,128 @@
...
-
-			<div class="m-2 tagWidget tagWidget-main p-4 text-center">
-				<h2 class="text-xl body-title pb-2 uppercase min-h-[4em]">{secondary.title}</h2>
-			</div>
-			<div class="m-2 tagWidget tagWidget-main p-4 text-center">
-				<p class="min-h-[12em] mt-2">
-					<!-- eslint-disable svelte/no-at-html-tags -->
-					{@html marked(secondary.content.replaceAll('<', '&lt;'))}
-				</p>
+<div class="relative mx-auto">
```
Comment (coyotte508) [inline src/lib/components/GalleryWidget/GalleryWidget.svelte:19]: Is this surrounding div necessary?

Comment (ithiame) [reply src/lib/components/GalleryWidget/GalleryWidget.svelte]: removed

**Review by coyotte508 — APPROVED** [review]

**Comment by Tirodem** [comment]
Good except < & > CTAs positions.
As seen on channel, put it on column title level, otherwise it's bothersome for users to have the CTAs to change their vertical position while sliding.

**Comment by Tirodem** [comment]
Make the text wider please :
image
After that it'll be good.

**Comment by Tirodem** [comment]
New try after commit & deploy :
image

**Pull request merged**